soccer/src/soccer/strategy/agent/position/position.hpp

```diff
@@ -229,6 +231,8 @@ class Position {
      */
     virtual void set_goalie_id(int goalie_id);
 
+    virtual void set_override_position(const strategy::OverridingPositions overriding_position){};
```

Comment:
oh this is ugly

we talked about this not blaming u but yes this confirms my vote to change AAC Position* into RFP* (or at that point just store the object, no extra indirection necessary)